projects/demo-maps/.browserslistrc

```diff
@@ -2,11 +2,16 @@
 # For additional information regarding the format and rule options, please see:
 # https://github.com/browserslist/browserslist#queries
 
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
-last 2 versions
+last 1 Chrome version
```

Comment:
Do we deliberately test only with latest Chrome, but with the two latest Safaris?

Reply:
This was generated by angular, I only want to update the File because we did not change anything on it.

Reply:
Roger that! No further questions.